Docker/docker-compose.autotls.yml

```
@@ -0,0 +1,65 @@
+    container_name: birdnet-go
+    restart: unless-stopped
+    ports:
+      - "80:80"      # Required for ACME HTTP-01 challenge
+      - "443:443"    # Required for HTTPS/AutoTLS
+    environment:
+      - TZ=${TZ:-UTC}
+      - BIRDNET_UID=${BIRDNET_UID:-1000}
+      - BIRDNET_GID=${BIRDNET_GID:-1000}
+      # Configure AutoTLS
+      - BIRDNET_SECURITY_AUTOTLS=true
+      - BIRDNET_SECURITY_HOST=${BIRDNET_HOST} # Required: Your domain name
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)